### ISSUE TITLE
'Can't separate key from value' when running breeze

### ISSUE THREAD

**Issue by oyarushe** [apache/airflow#16949]
**Apache Airflow version**: the lasters version from main ([4d8fd37](https://github.com/apache/airflow/commit/4d8fd37b6d110b348f29e677eaaaf9e447c837c7))

**Environment**:

- **Cloud provider or hardware configuration**:
- **OS** (e.g. from /etc/os-release): macOS Big Sur
- **Kernel** (e.g. `uname -a`): Darwin Kernel Version 20.4.0
- **Docker Desktop**: 3.5.2
- **Docker Compose**: v2.0.0-beta.6

**What happened**:

```
$ ./breeze -v
Executing script:

/Users/oyarush/projects/airflow/airflow/.build/dc_ci run --service-ports --rm airflow

++ dirname /Users/oyarush/projects/airflow/airflow/.build/dc_ci
+ cd /Users/oyarush/projects/airflow/airflow/.build
+ export HOST_USER_ID=502
...
+ USE_AIRFLOW_VERSION=
+ export USE_PACKAGES_FROM_DIST=false
+ USE_PACKAGES_FROM_DIST=false
+ export EXECUTOR=KubernetesExecutor
+ EXECUTOR=KubernetesExecutor
+ docker-compose run --service-ports --rm airflow
Can't separate key from value

ERROR: The previous step completed with error. Please take a look at output above 

###########################################################################################
                   EXITING WITH STATUS CODE 15
###########################################################################################

```

It only happens for docker-compose v2.0. So possible resolution can be to disable the experimental feature "Use Docker Compose V2"

**Comment by potiuk** [comment]
Fixed by #16950

**Comment by potiuk** [comment]
@oyarushe - fix to that problem have been implemented in https://github.com/docker/compose-cli/pull/2002 and released in 
https://github.com/docker/compose-cli/releases/tag/v2.0.0-rc.1  - maybe you could see if your Docker Desktop is updated to it and you could test bringing back the change.
